### ISSUE TITLE
Potential naming inconsistencies between ntc-templates and netmiko for fortinet devices

### ISSUE THREAD

**Issue by corvese** [networktocode/ntc-templates#534]
##### ISSUE TYPE
 - Bug Report

##### TEMPLATE USING

```
fortinet_fortios_get_router_info_bgp_summary.template
```

##### SAMPLE COMMAND OUTPUT

```
"get router info bgp summary"
```

##### SUMMARY

- I'm trying to obtain structured data from the aforementioned template using Netmiko
- I am returned unstructured data (the raw output)
- I suspect this is related to the naming convention used by this project: 
```
{{ vendor_os }}_{{ command_with_underscores }}.template
```
- Netmiko device_type is just ```fortinet```
- When I change the index file to use ```fortinet```and the template file name to ```fortinet_get_router_info_bgp_summary.template```I am able to obtain structured data without issue.

When using the following I am able to obtain structured data:
```fortinet_get_router_info_bgp_summary.template, .*, fortinet, g[[et]] r[[outer]] info bg[[p]] su[[mmary]]```

**Comment by jmcgill298** [comment]
@ccorvese This can be reverted back to `fortinet`, but the tests will have to be updated to not be dependent on the vendor/os distinction

**Comment by ccorvese** [comment]
> 
> 
> @ccorvese This can be reverted back to `fortinet`, but the tests will have to be updated to not be dependent on the vendor/os distinction

Thanks @jmcgill298. So would this proposed change to the automated tests be required for all vendors, or just for fortinet?

**Comment by jmcgill298** [comment]
I haven't had time to look at it in detail, but there should be a generic fix that just doesn't depend on parsing the names

**Comment by wmclendon** [comment]
this is still an issue, and I suspect why there are not many Fortinet textFSM templates published.  I have one for ARP that i am able to use now successfully with Netpalm and netmiko by loading the template manually into the collection with name just 'fortinet' instead of 'fortinet_fortios' that will unfortunately not work in this collection under the current naming scheme.  I'm not familiar with the testing mechanisms employed how to reduce the naming requirements at least for fortinet to be inline with netmiko

Thanks,

Will

**Comment by jmcgill298** [comment]
https://github.com/networktocode/ntc-templates/pull/831

**Comment by jmcgill298** [comment]
@wmclendon this should be fixed, and you can look at #831 for what is required for contributing back to this project. If you have any questions, please ask at networktocode.slack.com
